tests/api/test_account.py

```diff
@@ -1288,7 +1312,39 @@ def test_set_password(user_api_client, customer_user):
 
 
 @patch("saleor.account.emails.send_password_reset_email.delay")
-def test_password_reset_email(
+def test_request_password_reset_email(
+    send_password_reset_mock, staff_api_client, customer_user, permission_manage_users
+):
+    query = """
+    mutation CustomerRequestPasswordReset($id: ID!) {
+        customerRequestPasswordReset(id: $id) {
+            errors {
+                field
+                message
+            }
+        }
+    }
+    """
+    user_id = graphene.Node.to_global_id("User", customer_user.id)
+    variables = {"id": user_id}
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_users]
+    )
+    content = get_graphql_content(response)
+    data = content["data"]["customerRequestPasswordReset"]
+    assert data == {"errors": []}
+    assert send_password_reset_mock.call_count == 1
+    args, kwargs = send_password_reset_mock.call_args
```

Comment:
Maybe you can use - https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_with